### ISSUE TITLE
Debian Wheezy odoo v8 db_name error

### ISSUE THREAD

**Issue by zeigerpuppy** [odoo/odoo#3326]
I have installed odoo and set up a database on my database server,

however, when odoo starts it tries to access the incorrect database name.

The database I have created is on host 10.0.1.10 and is called "odoo".  I can connect to it from the terminal on the odoo server.

But as you can see from the log, odoo is trying to access a database called "postgres", not "odoo".
So postgres correctly rejects access.
```
    OperationalError: FATAL:  no pg_hba.conf entry for host "10.0.1.10", user "odoo", database "postgres", SSL on

    FATAL:  no pg_hba.conf entry for host "10.0.1.10", user "odoo", database "postgres", SSL off
```
I have set up my config correctly as far as I can see, but odoo seems to ignore the db_name option.

any ideas?
```
    [options]
    ; This is the password that allows database operations:
    admin_passwd = mypassword
    db_host = 10.0.1.10
    db_name = odoo
    db_port = 5432
    db_user = odoo
    db_password = mydbpassword
    addons_path = /usr/lib/python2.7/dist-packages/openerp/addons
```

**Comment by tommyma** [comment]
@zeigerpuppy , did you fix the issue? I met same issue as you said exactly, and have no idea to fix it.

I am using the package in source code , and using the space in the webfaction.com, so have no permission to create the database "postgres" to skip the checking of system.

**Comment by zeigerpuppy** [comment]
I did manage to fix the issue, I purged everything and tried with a local install and it threw an error.
Looks like my locales were not configured properly and the postgres database was using the wrong encoding (needs to be UTF).
So I updated the encoding on postgres and ran it again, worked this time.
However, after testing I have moved from odoo to taiga.io as my users preferred it and it works better on mobile devices.
